schema/schema.go

```diff
@@ -109,7 +109,7 @@ func ReadMerged() (*serializers.Schema, error) {
 		if fi.Mode()&os.ModeSymlink != 0 {
 			path, err = os.Readlink(path)
 			if err != nil {
-				return nil, err
```

Comment:
some day it might be nice to have this server have a logger and log that this happened somewhere so that a thoughtful dev could fix this :P